tools/vtx_extract.mk

```diff
@@ -0,0 +1,6 @@
+CPP := g++
+
+CPPFLAGS := -fpermissive
+
+vtx_extract: vtx_extract.cpp libmio0.c libmio0.h utils.c utils.h
```

Comment:
If this doesn't build when you run `make` can we instead integrate this into the normal tools makefile.

Ie. I think the program should be built by default. But only ran if the user desires to do so.

Honestly, it's probably fine if it runs automatically. I'll leave that decision up to you.